doc/api/async_hooks.md

```diff
@@ -730,7 +730,7 @@ class DBQuery extends AsyncResource {
 }
 ```
 
-#### Static method: `AsyncResource.bind(fn[, type])`
+#### Static method: `AsyncResource.bind(fn[, type, [thisArg]])`
```

Comment:
I think the YAML section should include some message about this change

Reply:
done!